### ISSUE TITLE
get_thumbnailer not working when using s3

### ISSUE THREAD

**Issue by philippeluickx** [SmileyChris/easy-thumbnails#364]
I have following code I am using in my REST serializer.

```python
            thumbnail_url = get_thumbnailer(obj._get_first_picture().original_image).get_thumbnail({
                'size': (1080, 720),
                'box': obj._get_first_picture().cropping,
                'crop': True,
                'detail': True,
                'upscale': True,
            }).url
```

For some reason, this is blocking the whole API on server-level. My guess is somewhere files are being referenced with a relative path (/media/...) and not a full path (aws url). Had a look at get_thumbnailer's source code
https://github.com/SmileyChris/easy-thumbnails/blob/714aa5f36894797ca6c980c9bc8cfa52a46156c1/easy_thumbnails/files.py#L20
and could not really figure things out...

I don't manage to debug this as it works fine locally and I am making some assumptions here.

Any pointers are appreciated!

Update:
So the get_thumbnailer seems not to be the main issue, as when I navigate to a particular object in my REST API, the url's are generated correctly.
It's only messed up in the list view... both using the same serializer.

The main reason I believe that the get_thumbnailer is the culprit is that when I simply return a string, things work correctly.

**Comment by philippeluickx** [comment]
Could it be that the REST api is not waiting long enough for all thumbnails to be generated in the list?
I am getting a 502 bad gateway error by the way...

**Comment by czpython** [comment]
Fixed by #369

**Comment by SmileyChris** [comment]
Thanks, release coming very soon!
